src/parser/test/ParserTest.cpp

```diff
@@ -386,6 +403,15 @@ TEST(Parser, InsertEdge) {
         auto result = parser.parse(query);
         ASSERT_TRUE(result.ok()) << result.status();
     }
+    // multi edge
+    {
+        GQLParser parser;
+        std::string query = "INSERT EDGE transfer(amount, time) "
```

Comment:
you can try to add a veryyyyyy loooooonnnnng version 1537408527111111111111111111111111111